src/windowsservercore/2004/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# escape=`
+FROM mcr.microsoft.com/windows/servercore:2004-amd64
+
+SHELL ["cmd", "/S", "/C"]
```

Comment:
(NIT) This isn't necessary as it is the default shell.

Reply:
done.